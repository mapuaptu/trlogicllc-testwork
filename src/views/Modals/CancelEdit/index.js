import Button from '@/components/Button/index.vue';

export default {
  name: 'ModalCancelEdit',
  components: {
    Button,
  },
  methods: {
    onNo() {
      return this.$emit('close');
    },
    onYes() {
      this.$emit('close');

      return this.$router.push({ name: 'home' });
    },
  },
};
