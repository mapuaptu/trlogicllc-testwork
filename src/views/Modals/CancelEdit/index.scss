@import '@/styles/_variables.scss';

.modal {
  position: relative;
}
