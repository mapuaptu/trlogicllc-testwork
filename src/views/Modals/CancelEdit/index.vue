<template>
  <div
    :class="$style.modal"
    class="modal"
  >
    <div class="modal-header">
      Cancel edit note
    </div>

    <div class="modal-content">
      Are you sure you want to cancel edit note
    </div>

    <div class="modal-footer">
      <Button
        :fill="true"
        @click.native="onNo"
      >
        No
      </Button>

      <Button @click.native="onYes">
        Yes
      </Button>
    </div>
  </div>
</template>

<script src="./index.js"></script>
<style src="./index.scss" lang="scss" module></style>
