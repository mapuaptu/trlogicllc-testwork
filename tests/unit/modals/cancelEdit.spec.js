import CancelEdit from '@/views/Modals/CancelEdit/index.vue';
import { mount } from '@vue/test-utils';

describe('Modal createNote component', () => {
  it('1. Rendered success', async () => {
    const wrapper = await mount(CancelEdit);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
