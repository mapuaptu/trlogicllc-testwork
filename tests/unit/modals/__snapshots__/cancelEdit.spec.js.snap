// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal createNote component 1. Rendered success 1`] = `
<div class="modal">
  <div class="modal-header">
    Cancel edit note
  </div>
  <div class="modal-content">
    Are you sure you want to cancel edit note
  </div>
  <div class="modal-footer">
    <div class=""><span>
      No
    </span>
      <!---->
    </div>
    <div class=""><span>
      Yes
    </span>
      <!---->
    </div>
  </div>
</div>
`;
